README.md

```diff
@@ -124,7 +124,7 @@ const blockLocal = await core.get(44, { wait: false })
 
 Check if the core has all blocks between `start` and `end`.
 
-#### `const updated = await core.update()`
+#### `const updated = await core.update([options])`
 
 Wait for the core to try and find a signed update to its length.
 Does not download any data from peers except for a proof of the new core length.
```

Comment:
Probably this two lines requires a rewrite?

Reply:
Nice, yes! Can you update? Ie waits for initial proof until all findingPeers calls has finished